camel-idea-plugin/src/main/java/com/github/cameltooling/idea/service/CamelCatalogService.java

```diff
@@ -46,10 +46,10 @@ public CamelCatalogService(Project project) {
         this.project = project;
         ApplicationManager.getApplication().getMessageBus()
             .connect(this)
-            .subscribe(CamelPreferenceService.CamelCatalogProviderChangeListener.TOPIC, this::onCamelCatalogProviderChanged);
+            .subscribe(CamelPreferenceService.CamelCatalogProviderChangeListener.TOPIC);
         project.getMessageBus()
             .connect(this)
-            .subscribe(CamelService.CamelCatalogListener.TOPIC, this::onCamelCatalogReady);
```

Comment:
ditto

Reply:
ok

Reply:
This was by mistake due to one additionl space this file got commited sorry for that